### PR TITLE
cmd/geth: add support for sepolia testnet

### PR DESCRIPTION
**Pull request by holiman**
This PR adds suppor for 'Sepolia', a new testnet to replace ropsten. It differs a bit from what's already running, since I noticed that the existing one has a genesis difficult of `0x200`, which is below the minimum of `0x2000`. When the difficulty is below that value, the YP difficulty calculation is ambiguous. 
Also, the existing one had empty extradata, which might cause issues, so I added some data there.

### CODE REVIEW

**Comment by garyschulte** [comment]
> Geth has a bug if it's mining on initial empty db and then syncs. Same happens on Clique signers too btw, there they just mess up block 1. I think I opened an issue a year or two ago about it. Miners need to sync first, then mine as a workaround.
> […](#)

FWIW, at the time I started the miner, the geth node had 33k blocks in its db already.

File: core/genesis.go [hunk 1]
```diff
@@ -400,6 +402,18 @@ func DefaultGoerliGenesisBlock() *Genesis {
 	}
 }
 
+func DefaultSepoliaGenesisBlock() *Genesis {
```
Comment (karalabe) [inline core/genesis.go:405]: `// DefaultSepoliaGenesisBlock returns the Sepolia network genesis block.`

**Review by karalabe — APPROVED** [review]
SGTM

File: params/config.go
Comment (Md0135) [inline params/config.go:1]: Belki de çalışmak zorunda kaldık

**Review by Hasmuni5757 — APPROVED** [review]

**Review by JTKO1 — APPROVED** [review]

File: core/genesis.go (hunk 1 above)
Comment (Khoshi76) [reply core/genesis.go:406]: Yes

**Pull request merged**